### ISSUE TITLE
insert works odd with js-false statements

### ISSUE THREAD

**Issue by 8026787** [sapienlab/dbh-pg#12]
```
this.insert('table', {hidden:true}); // ok
```
```
this.insert('table', {hidden:false}); 
// error: value in column "hidden" violates not-null constraint
```
```
this.insert('table', {hidden:'false'}); // ok
```
```
var x = 0;
this.insert('table', {value:x});
// error: null value in column "value" violates not-null constraint
````
````
var x = 0;
this.insert('table', {value:x+''}); // ok
````

**Comment by roro89** [comment]
Hi, Thank you for reporting this error!
I see that this is a bug in [lib/Connection.js#L142](https://github.com/sapienlab/dbh-pg/blob/master/lib/Connection.js#L142).

I'll fix it within hours.

**Comment by roro89** [comment]
This bug should be solved with the [v2.1.0](https://github.com/sapienlab/dbh-pg/pull/15)

Please, comment if is Ok to close this issue :)

**Comment by 8026787** [comment]
It's Ok, thank you )
